endoreg_db/views/requirement/lookup.py

```
@@ -0,0 +1,92 @@
+        pe_id = request.data.get("patient_examination_id")
+        if not pe_id:
+            return Response({"detail": "patient_examination_id required"}, status=status.HTTP_400_BAD_REQUEST)
+        try:
+            token = create_lookup_token_for_pe(int(pe_id))
+            print(f"Created lookup token {token} for PE {pe_id}")
+        except Exception as e:
+            return Response({"detail": str(e)}, status=status.HTTP_400_BAD_REQUEST)
+        return Response({"token": token}, status=status.HTTP_201_CREATED)
+
+    @action(detail=True, methods=["get"], url_path="all")
+    def get_all(self, request, pk=None):
```

Ruff (0.12.2)

Unused method argument: `request`

(ARG002)